ostelco-ios-client/Coordinators/EKYC/JumioCoordinator.swift

```diff
@@ -92,7 +90,7 @@ class JumioCoordinator: NSObject {
         config.delegate = self
         
         // This must be a 3-letter code following ISO-3166
-        config.preselectedCountry = country.threeLetterCountryCode
+        config.preselectedCountry = Country(regionID).threeLetterCountryCode
```

Comment:
@sgoodwin this is not going to fly, regionID is not equal to country code, this has to be reverted else we can get issues in the future, when we get a region with ID that does not resemble a country code.